gui/default/syncthing/core/syncthingController.js

```diff
@@ -1226,6 +1227,9 @@ angular.module('syncthing.core')
 
         $scope.saveDevice = function () {
             $('#editDevice').modal('hide');
+            if ($scope.currentDevice.deviceID === '') {
+                $scope.currentDevice.deviceID = $scope.manualDeviceID;
+            }
```

Comment:
I don't want to display a text input field unless "Enter manually below" is selected. its ugly(er) and then the user could theoretically both choose an ID from the list and enter one below.